<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadPreviewDialog</class>
 <widget class="QDialog" name="LoadPreviewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="itemListTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="contentEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>451</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>460</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>460</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="selectAllBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>select all</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
